app/views/layouts/application.html.erb

```diff
@@ -9,20 +9,15 @@
     <meta name="mobile-web-app-capable" content="yes">
     <%= favicon_link_tag "icons/apple-icon-180x180", rel: "icon", type: "image/png" %>
     <link rel="manifest" href="<%= asset_path 'manifest.json' %>">
-
-
     <link href="https://api.mapbox.com/mapbox-gl-js/v2.11.0/mapbox-gl.css" rel="stylesheet">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/flatpickr/dist/flatpickr.min.css">
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
-
   <body>
     <%= render "shared/navbar" %>
     <%= render "shared/flashes" %>
-    <div class="container">
-      <%= yield %>
-    </div>
+    <%= yield %>
```

Comment:
I removed the wrap around yield, while adding .container around yield is best practice, our map requires a clean edge and the map to extend to the edge of the page. Because of this we will need to wrap our pages with a .container if needed.